src/api/json/catalog.json

```
@@ -4868,6 +4868,12 @@
         "1.0": "https://json-schema.api.strmprivacy.io/latest/strmprivacy.api.entities.v1.DataContract.json"
       }
     },
+    {
+      "description": "Sublime syntax",
```

The description is currently the same as the name, i.e. not descriptive...
```suggestion
      "description": "Sublime Text/Merge syntax highlighting definition",
```